yt/chyt/controller/internal/chyt/controller.go

```diff
@@ -32,11 +32,13 @@ type Config struct {
 	AddressResolver           map[string]any       `yson:"address_resolver"`
 	EnableYandexSpecificLinks *bool                `yson:"enable_yandex_specific_links"`
 	ExportSystemLogTables     *bool                `yson:"export_system_log_tables"`
+	DefaultEnableGeoData      *bool                `yson:"default_enable_geo_data"`
```

Comment:
The naming is inconsistent here.

It should be either just `EnableGeoData` (like `ExportSystemLogTables` option above), or we need to change the naming of the `ExportSystemLogTables` as well

Reply:
Oh, I got the idea. Fixed.